therapy/__init__.py

```diff
@@ -38,12 +38,8 @@ def __init__(self, *args, **kwargs):
                     for k, v in SourceIDAfterNamespace.__members__.items()
                     if v.value != ''}
 
-# Sources that are not allowed in normalize endpoint due to license
-PROHIBITED_SOURCES = {s.value.lower()
-                      for s in ProhibitedSources.__members__.values()}
-
 # Sources that are allowed in normalize endpoint
-ACCEPTED_SOURCES = SOURCES.keys() - PROHIBITED_SOURCES
+ACCEPTED_SOURCES = SOURCES.keys()
```

Comment:
We can probably drop this now (I think -- double check me -- that `XREF_SOURCES` would accomplish the same thing)